From 4816a705e7917ee58d3972fefe163189eb412d36 Mon Sep 17 00:00:00 2001
From: Emekcan Aras <emekcan.aras@arm.com>
Date: Fri, 16 Jun 2023 10:47:48 +0100
Subject: [PATCH] plat: corstone1000: add compile definitions for
 ECP_DP_SECP512R1

Corstone1000 runs PSA-API tests which requires this ECC algorithm.
Without setting this, corstone1000 fails psa-api-crypto-test no 243.

Signed-off-by: Emekcan Aras <emekcan.aras@arm.com>
Upstream-Status: Pending
---
 platform/providers/arm/corstone1000/platform.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/platform/providers/arm/corstone1000/platform.cmake b/platform/providers/arm/corstone1000/platform.cmake
index a3c4209b..ff044ed7 100644
--- a/platform/providers/arm/corstone1000/platform.cmake
+++ b/platform/providers/arm/corstone1000/platform.cmake
@@ -13,3 +13,5 @@ target_compile_definitions(${TGT} PRIVATE
 	SMM_VARIABLE_INDEX_STORAGE_UID=0x787
 	SMM_GATEWAY_MAX_UEFI_VARIABLES=100
 )
+
+add_compile_definitions(MBEDTLS_ECP_DP_SECP521R1_ENABLED)
-- 
2.25.1

